LidarrAPI/Database/Migrations/20180201022530_AddStatusField.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+using System.Collections.Generic;
+
+namespace LidarrAPI.Database.Migrations
+{
+    public partial class AddStatusField : Migration
```

Comment:
Should this have some type of identifier for ordering migrations? Or is that handled by the migration attribute of the `Designer.cs` file?

Reply:
This should be tracked by the designer and there is a migration table in the DB .. that shows order of migrations :)